src/main/java/org/team1294/firstpowerup/robot/commands/AutoDeliverCrateCommand.java

```diff
@@ -0,0 +1,9 @@
+package org.team1294.firstpowerup.robot.commands;
+
+import edu.wpi.first.wpilibj.command.CommandGroup;
+
+public class AutoDeliverCrateCommand extends CommandGroup {
+    public void setEnabled(final boolean enabled) {
```

Comment:
What's the point of implementing (overriding?) this?

Reply:
The AutoSidePositionCommand adds a sequence of commands. During the initialize, GameSpecificMessage is analyzed and different angles and distances are used. In the case that all we can do is cross the auto line, we dont want to deliver the crate to the floor. TBD if this is the right implementation but this was a placeholder for the idea.